src/components/Login.tsx

```diff
@@ -0,0 +1,82 @@
+import styled from 'styled-components/macro'
+import heroImg from '../assets/images/Login-hero.jpg'
+import Button from './common/Button/Button'
+
+const Login = () => {
+  const authEndpoint: string = 'https://accounts.spotify.com/authorize'
+  const redirectUri: string = 'http://localhost:3000/'
+  const clientId: string = 'ce11cc084edb46788a28b35de4f32f65'
```

Comment:
`ce11cc084edb46788a28b35de4f32f65` should be private (load it from environment variable?)